doc/how-to/initialise.md

```diff
@@ -30,7 +30,7 @@ Complete the following steps to initialise MicroCloud:
 
 1. Select whether you want to set up more than one machine.
 
-   This allows you to create a MicroCloud using a single node.
+   This allows you to create a MicroCloud using a single cluster member.
```

Comment:
In the docs we have node, cluster member and machine used on a regular basis.
Would it be the idea to start sticking with one of them?

Reply:
Yes, discussed and approved by LXD management team, to reduce confusion where the words `node`, `cluster member`, and `host` are used interchangeably & inconsistently. `machine` can also be a culprit, but it does not need to be replaced in some cases, as in `virtual machine` or `For production environments, a minimum of 3 physical machines is required as cluster members.` (You wouldn't say "a minimum of 3 cluster members is required as cluster members."

Let's use `cluster member` going forward for `node`, `member`, `host`, as well as `machine` if it makes sense. `member(s)` can be used if there's a clear antecedent (e.g. "You can use the `microcloud cluster` command to show information about the cluster members, or to remove specific **members**."). LXD's documentation already conforms to this, but MicroCloud's is inconsistent presently.

Reply:
Thanks for the clarification, sounds good to me.